st2common/st2common/util/virtualenvs.py

```diff
@@ -99,7 +100,14 @@ def setup_pack_virtualenv(pack_name, update=False, logger=None, include_pip=True
         install_requirement(virtualenv_path=virtualenv_path, requirement=requirement,
                             proxy_config=proxy_config, logger=logger)
 
-    # 3. Install pack-specific requirements
+    # 3. Install base Python 3 requirements which are common to all the packs
+    if use_python3:
+        logger.debug('Installing base Python 3requirements')
```

Comment:
Typo, should be `Installing base Python 3 requirements`.

Reply:
Thanks, fixed.